tests/test_bic.py

```diff
@@ -23,23 +23,34 @@ def test_bic_no_branch_code():
     assert bic.formatted == 'GENO DE M1'
 
 
-def test_country_bank_code():
-    assert BIC('ABNAJPJTXXX').country_bank_code is None
-    assert BIC('GENODEM1GLS').country_bank_code == '43060967'
-
-
 def test_bic_properties():
     bic = BIC('GENODEM1GLS')
     assert bic.length == 11
     assert bic.bank_code == 'GENO'
-    assert bic.branch_code == 'GLS'
     assert bic.country_code == 'DE'
     assert bic.location_code == 'M1'
+    assert bic.branch_code == 'GLS'
     assert bic.country_bank_code == '43060967'
+    assert bic.bank_name == 'GLS Gemeinschaftsbank'
+    assert bic.bank_short_name == 'GLS Gemeinschaftsbk Bochum'
     assert bic.exists
     assert bic.type == 'passive'
 
 
+def test_unknown_bic_properties():
+    bic = BIC('ABNAJPJTXXX')
+    assert bic.length == 11
+    assert bic.bank_code == 'ABNA'
+    assert bic.country_code == 'JP'
+    assert bic.location_code == 'JT'
+    assert bic.branch_code == 'XXX'
+    assert bic.country_bank_code == None
```

Comment:
Please use `is` for boolean expressions that compare to `None`, `True`, `False` and so on. This is also the reason why the tests fail.